incubator/kubirds-datadog-reactor/templates/reactor.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kubirds.com/v1
+kind: Reactor
+metadata:
+  name: {{ include "kubirds-datadog-reactor.fullname" . }}
+  labels:
+    {{- include "kubirds-datadog-reactor.labels" . | nindent 4 }}
+    {{ if .Values.labels }}
+    {{ .Values.labels | toYaml | nindent 4 }}
+    {{ end}}
+spec:
```

Comment:
New in Kubirds 2.0.0-alpha1:

```yaml
serviceAccountName: {{ include "kubirds-datadog-reactor.serviceAccountName" . }}
```

Every unit and reactor now has a service account defaulting to `default`. The helm chart should override this.